harness/determined/searcher/core_search_runner.py

```diff
@@ -99,9 +101,21 @@ def pause_searcher(self, session: client.Session) -> bool:
         assert self.info is not None
 
         exp_id = self.info.trial.experiment_id
-        bindings.post_PauseExperiment(session, id=exp_id)
+        post_PauseExperiment(session, id=exp_id)
         while self.context.preempt.should_preempt() is False:
             time.sleep(5)
             continue
 
         return True
+
+
+def _is_experiment_active(state: determinedexperimentv1State) -> bool:
+    if (
+        state == determinedexperimentv1State.STATE_ACTIVE
+        or state == determinedexperimentv1State.STATE_QUEUED
+        or state == determinedexperimentv1State.STATE_RUNNING
+        or state == determinedexperimentv1State.STATE_STARTING
+        or state == determinedexperimentv1State.STATE_PULLING
+    ):
+        return True
+    return False
```

Comment:
PP:
```suggestion
    return (
        state == determinedexperimentv1State.STATE_ACTIVE
        or state == determinedexperimentv1State.STATE_QUEUED
        or state == determinedexperimentv1State.STATE_RUNNING
        or state == determinedexperimentv1State.STATE_STARTING
        or state == determinedexperimentv1State.STATE_PULLING
    )
```

Reply:
Just found this: https://github.com/determined-ai/determined/blob/3752641c601afb20d548a75779d4348d94e7c620/master/internal/api_experiment.go#L115 🤦‍♀️ 
will update with that